<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.50" />
  <meta name="author" content="John D. Storey">
  <meta name="description" content="Statistical Genomics Laboratory">

  
  
  
    
  
  
  <link rel="stylesheet" href="/css/bootstrap.min.css">
  <link rel="stylesheet" href="/css/font-awesome.min.css">
  <link rel="stylesheet" href="/css/academicons.min.css">
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  <link rel="alternate" href="http://storeylab.org/software/index.xml" type="application/rss+xml" title="Storey Lab">
  <link rel="feed" href="http://storeylab.org/software/index.xml" type="application/rss+xml" title="Storey Lab">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="http://storeylab.org/software/">

  

  <title>Software Packages | Storey Lab</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">Storey Lab</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/">
            
            <span>About</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/team/">
            
            <span>Members</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/publication/">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/software/">
            
            <span>Software</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/teaching/">
            
            <span>Teaching</span>
          </a>
        </li>

        
        

      </ul>

    </div>
  </div>
</nav>




<div class="container">
  <div class="row">
    <div class="col-md-12">
      <h1>Software Packages</h1>
      

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-biobroom"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> biobroom</h2>

<p>This R package converts standard objects constructed by bioinformatics packages, especially those in Bioconductor, to the tidy data format.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/biobroom.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/biobroom" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-dnamix"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> dnamix</h2>

<p>This Fortran program calcluates likelihood ratios as they pertain to mixed DNA samples encountered in forensic science.
<br>
<a href="https://github.com/StoreyLab/dnamix" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-edge"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> edge</h2>

<p>This R package implements methods for carrying out differential expression analyses of genome-wide gene expression studies.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/edge.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/edge" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-eigenr2"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> eigenR2</h2>

<p>This R package calculates a high-dimensional version of the classic $R^2$ statistic.
<br>
<a href="https://github.com/StoreyLab/eigenR2" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-gcatest"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> gcatest</h2>

<p>This R package implements the genotype-conditional association test (GCATest), which is an association test for genome wide association studies that controls for population structure under a general class of trait models.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/gcatest.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-jackstraw"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> jackstraw</h2>

<p>This R package calculates a ignificance test for association between variables and their estimated latent variables. Latent variables may be estimated by principal component analysis (PCA), logistic factor analysis (LFA), and other techniques.
<br>
<a href="https://cran.r-project.org/web/packages/jackstraw/index.html" class="btn btn-primary btn-outline btn-xs" target="_blank">CRAN</a>
<a href="https://github.com/StoreyLab/gcatest" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-lfa"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> lfa</h2>

<p>This R package implements logistic factor analysis, which is a PCA analogue on Binomial data via estimation of latent structure in the natural parameter space.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/lfa.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/lfa" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-qvalue"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> qvalue</h2>

<p>This R package takes a list of p-values resulting from the simultaneous testing of many hypotheses and estimates their q-values and local FDR values.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/qvalue.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/qvalue" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-snm"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> snm</h2>

<p>This R package performs a supervised normalization of microarray data that takes into account the study design.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/snm.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-subseq"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> subSeq</h2>

<p>This R package performs subsampling of high throughput sequencing count data for use in experimental design and analysis.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/subSeq.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/subSeq" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-sva"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> sva</h2>

<p>This R package performs surrogate variable analysis to account for sources of systematic variation not included in the study design model.
<br>
<a href="http://www.bioconductor.org/packages/release/bioc/html/sva.html" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/jtleek/sva-devel" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-terastructure"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> terastructure</h2>

<p>This C++ program fits the admixture model on tera-sample-sized data sets (~$10^{12}$ observed genotypes). This package provides a scalable, multi-threaded implementation that can be run on a single computer.
<br>
<a href="https://github.com/StoreyLab/terastructure" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

<h2 id="i-class-fa-fa-code-aria-hidden-true-style-color-03396c-font-size-80-i-trigger"><i class="fa fa-code" aria-hidden="true" style="color:#03396c;font-size:80%"></i> trigger</h2>

<p>This R package provides tools for the statistical analysis of genetics of gene expression data.
<br>
<a href="http://bioconductor.org/packages/trigger/" class="btn btn-primary btn-outline btn-xs" target="_blank">Bioconductor</a>
<a href="https://github.com/StoreyLab/trigger-devel" class="btn btn-primary btn-outline btn-xs" target="_blank">GitHub</a></p>

    </div>
  </div>
</div>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2017 John D. Storey &middot; 

      Website design derived from the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.4/TweenMax.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/latest/plugins/ScrollToPlugin.min.js"></script>
    <script src="/js/jquery-1.12.3.min.js"></script>
    <script src="/js/bootstrap.min.js"></script>
    <script src="/js/isotope.pkgd.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.1/imagesloaded.pkgd.min.js"></script>
    <script src="/js/hugo-academic.js"></script>
    

    

    
    

    
    
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
    </script>
    <script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML"></script>
    

  </body>
</html>

