<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.20.7" />
  <meta name="author" content="John D. Storey">
  <meta name="description" content="Statistical Genomics Laboratory">

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/storeylab.org/css/highlight.min.css">
    
  
  <link rel="stylesheet" href="/storeylab.org/css/bootstrap.min.css">
  <link rel="stylesheet" href="/storeylab.org/css/font-awesome.min.css">
  <link rel="stylesheet" href="/storeylab.org/css/academicons.min.css">
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/storeylab.org/css/hugo-academic.css">
  

  <link rel="alternate" href="storeylab.org/index.xml" type="application/rss+xml" title="Storey Lab">
  <link rel="feed" href="storeylab.org/index.xml" type="application/rss+xml" title="Storey Lab">

  <link rel="icon" type="image/png" href="/storeylab.org/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/storeylab.org/img/apple-touch-icon.png">

  <link rel="canonical" href="storeylab.org/publication/woo_2010/">

  

  <title>A computationally efficient modular optimal discovery procedure | Storey Lab</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/storeylab.org/">Storey Lab</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/storeylab.org">
            
            <span>About</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/storeylab.org/team/">
            
            <span>Members</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/storeylab.org/publication/">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/storeylab.org/software/">
            
            <span>Software</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/storeylab.org/teaching/">
            
            <span>Teaching</span>
          </a>
        </li>

        
        

      </ul>

    </div>
  </div>
</nav>

<div class="pub" itemscope itemtype="http://schema.org/CreativeWork">

  

  <div class="container pub-title">
    <h1 itemprop="name">A computationally efficient modular optimal discovery procedure</h1>
    <span class="pub-authors" itemprop="author">
      
      S. Woo, J. T. Leek, John D. Storey
      
    </span>
    <span class="pull-right">
      

    </span>
  </div>

</div>
<div class="article-container">

  

  <h3>Abstract</h3>
  <p class="pub-abstract" itemprop="text">Motivation: It is well known that patterns of differential gene expression across biological conditions are often shared by many genes, particularly those within functional groups. Taking advantage of these patterns can lead to increased statistical power and biological clarity when testing for differential expression in a microarray experiment. The optimal discovery procedure (ODP), which maximizes the expected number of true positives for each fixed number of expected false positives, is a framework aimed at this goal. Storey et al. introduced an estimator of the ODP for identifying differentially expressed genes. However, their ODP estimator grows quadratically in computational time with respect to the number of genes. Reducing this computational burden is a key step in making the ODP practical for usage in a variety of high-throughput problems. Results: Here, we propose a new estimate of the ODP called the modular ODP (mODP). The existing &#39;full ODP&#39; requires that the likelihood function for each gene be evaluated according to the parameter estimates for all genes. The mODP assigns genes to modules according to a Kullbackâ€“Leibler distance, and then evaluates the statistic only at the module-averaged parameter estimates. We show that the mODP is relatively insensitive to the choice of the number of modules, but dramatically reduces the computational complexity from quadratic to linear in the number of genes. We compare the full ODP algorithm and mODP on simulated data and gene expression data from a recent study of Morrocan Amazighs. The mODP and full ODP algorithm perform very similarly across a range of comparisons.</p>

  

  <div class="row">
    <div class="col-sm-1"></div>
    <div class="col-sm-10">
      <div class="row">
        <div class="col-xs-12 col-sm-3 pub-row-heading">Publication</div>
        <div class="col-xs-12 col-sm-9"><em>Bioinformatics</em>, 27   (4): 509&ndash;515
        </div>
      </div>
    </div>
    <div class="col-sm-1"></div>
  </div>
  <div class="visible-xs space-below"></div>

  
  <div class="row">
    <div class="col-sm-1"></div>
    <div class="col-sm-10">
      <div class="row">
        <div class="col-xs-12 col-sm-3 pub-row-heading">DOI</div>
        <div class="col-xs-12 col-sm-9" itemprop="doi">
          10.1093/bioinformatics/btq701
        </div>
      </div>
    </div>
    <div class="col-sm-1"></div>
  </div>
  <div class="visible-xs space-below"></div>
  

  <div class="row">
    <div class="col-sm-1"></div>
    <div class="col-sm-10">
      <div class="row">
        <div class="col-xs-12 col-sm-3 pub-row-heading">Date</div>
        <div class="col-xs-12 col-sm-9" itemprop="datePublished">
          2010-12-24
        </div>
      </div>
    </div>
    <div class="col-sm-1"></div>
  </div>
  <div class="visible-xs space-below"></div>

  <div class="row" style="padding-top: 10px">
    <div class="col-sm-1"></div>
    <div class="col-sm-10">
      <div class="row">
        <div class="col-xs-12 col-sm-3 pub-row-heading" style="line-height:34px;">Links</div>
        <div class="col-xs-12 col-sm-9">

          




<a class="btn btn-primary btn-outline" href="https://dx.doi.org/10.1093/bioinformatics/btq701" target="_blank">
  Web
</a>










        </div>
      </div>
    </div>
    <div class="col-sm-1"></div>
  </div>
  <div class="visible-xs space-below"></div>

  <div class="space-below"></div>

  <div class="article-style"></div>

</div>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2017 John D. Storey &middot; 

      Website design derived from the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.4/TweenMax.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/latest/plugins/ScrollToPlugin.min.js"></script>
    <script src="/storeylab.org/js/jquery-1.12.3.min.js"></script>
    <script src="/storeylab.org/js/bootstrap.min.js"></script>
    <script src="/storeylab.org/js/isotope.pkgd.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.1/imagesloaded.pkgd.min.js"></script>
    <script src="/storeylab.org/js/hugo-academic.js"></script>
    

    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
    </script>
    <script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML"></script>
    

  </body>
</html>

